{"name": "test_Wijken_failure", "status": "passed", "description": "Failure API request for the Wijken", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/wijken/0363097000000C to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1647269866188, "stop": 1647269866313}], "start": 1647269866188, "stop": 1647269866313, "uuid": "c5579dcc-aff1-4a44-9817-be48804f6157", "historyId": "037430a115a2552756c28c3061e688ed", "testCaseId": "42c285f4cbf7c1d3064bb577bde9fffe", "fullName": "e2etest.test_datasetgebiedenwijken#test_Wijken_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenwijken"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "85329-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenwijken"}]}